docs/ota-client-guide/modules/ROOT/pages/client-provisioning-methods.adoc

```diff
@@ -14,63 +14,58 @@ endif::[]
-Once a device boots up, it connects to the OTA Connect server and provides the pre-installed device certificate to verify the device identity. The OTA Connect server then verifies that the certificate came from your CA.
-With this method, it is extremely difficult for an unauthorized device to join your fleet.
+In this method there is no shared credential. Instead the following steps occur:
 
+* You provide us with your own fleet root CA’s certificate (but NOT the private key).
+* Then, you make sure that each device acquires, in some out-of-band process, an X.509 certificate signed by your fleet root CA. In an ideal world, you would generate a keypair and a self-signed X.509 certificate inside an HSM on the device (so that there’s never any private key material outside of the HSM), then submit a CSR for that self-signed device certificate using your own existing PKI (e.g. via PKCS#10), and an authentication method of your choosing. (In an automotive OEM context, for example, that might be a private server inside the factory infrastructure, using the physical location and an airgapped network as ways to authenticate the request’s validity.)
+* Once the device has its signed certificate, it can register with the OTA Connect server: we simply verify that your fleet root CA has signed the vehicle certificate, and use the CNI on the vehicle certificate to identify it. If a device with that name already exists, it’s a normal update transaction; If not, we register it as a new vehicle.
 
-The following diagram summarizes the differences in how devices are provisioned between the two methods.
+Note that nowhere in this process have we had to use a shared credential, nor has any private key material existed outside of the vehicle (for vehicle keys), or your own PKI (for the fleet root CA). We have also used mutual TLS for transport security right from the beginning. This is why we describe this process as the more secure option.
 
-.How the server roles change depending on which method you choose:
-[caption="Figure 2: "]
-image::prov-diff-devices.png[]
 
-With "shared credential" provisioning, devices identified by the provisioning key and the device ID. The role of "issuer" is played by the OTA Connect server, which expects a provisioning key.
-With "device credential" provisioning, you decide how devices are identified. You could generate and preinstall certificate signing request (CSR) files which provide your Certificate Authority (CA) with all the necessary identification details.
+With "shared credential" provisioning, devices are identified by the provisioning key. The role of the OTA server is to generate a keypair and an X.509 certificate, and sign off with the fleet root CA.
+With "device credential" provisioning, you decide how devices are identified. You provide the OTA server with only your fleet root CA and ensure your devices already have an X.509 certificate signed by your fleet root CA.
```

Comment:
Is it intentional to have these two bits on separate lines without an empty line in between? I don't have strong feelings here, but it will be rendered as one longer paragraph.

Reply:
@patrickvacek yes you are right. I didn't realize it would be rendered in the same paragraph. I'll make the change.